semver/difference.ts

```diff
@@ -1,32 +1,28 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 import type { ReleaseType, SemVer } from "./types.ts";
-import { eq } from "./eq.ts";
+import { compareIdentifier } from "./_shared.ts";
 
-/** Returns difference between two versions by the release type, or
- * `undefined` if the versions are the same. */
+/**
+ * Returns difference between two versions by the release type, or `undefined` if the versions are the same.
+ */
 export function difference(
-  s0: SemVer,
-  s1: SemVer,
+  version1: SemVer,
+  version2: SemVer,
```

Comment:
IMO, `s0` and `s1` were fine as they were. Was there something about them that was unclear?

Reply:
I think we should not have to abbreviate parameter names. While the code gets a bit bigger, full names are easier to comprehend in context imo.

Reply:
`s0` `s1` look fine to me too as the scope of them are enough short here. I think there's very little chance for them to be confused with something else.

Reply:
ok, reverted.